hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3APrefetchingInputStream.java

```diff
@@ -60,8 +61,7 @@ public ITestS3APrefetchingInputStream() {
     super(true);
   }
 
-  private static final Logger LOG =
-      LoggerFactory.getLogger(ITestS3APrefetchingInputStream.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ITestS3APrefetchingInputStream.class);
```

Comment:
not needed; just revert to make patch smaller. unless you really, really want this